test/integration/open-with/OpenWith.int.js

```diff
@@ -0,0 +1,142 @@
+// <reference types="Cypress" />
+import l from '../../support/i18n';
+
+describe('OpenWith', () => {
+    beforeEach(() => {
+        cy.server();
+        cy.route('GET', '**/files/*?fields=extension', 'fixture:open-with/file-extension-document');
+    });
+
+    it('A single integration', () => {
+        cy.route('GET', '**/files/*/open_with_integrations', 'fixture:open-with/integration-adobe');
+        cy.visit('/ContentOpenWith');
+
+        // The button should be enabled
+        cy.getByTestId('singleintegrationbutton').should('not.have.attr', 'aria-disabled', 'true');
```

Comment:
consider adding constants for all these test ids that get used in many places

Reply:
I think it's actually more efficient to use helper methods, like: `const getSingleButton = () => cy.getByTestId('singleintegrationbutton');`

Reply:
yup that makes more sense